lib/http.ex

```diff
@@ -0,0 +1,18 @@
+defmodule UncookedGps.Http do
+  require Logger
+
+  def user_agent do
+    team_email = Application.fetch_env!(:uncooked_gps, :team_email)
+    "UncookedGps/0.1.0 (#{team_email})"
+  end
+
+  def get(url) do
+    Req.get!(url, user_agent: user_agent())
+    Logger.info("get url=#{url} agent=#{user_agent()}")
```

Comment:
question: this looks like it's in the wrong order. since `Logger.info` is last, the function is return the result of the logger call, not the HTTP query. Also makes me think that something in the test cases is missing if it didn't check this.

Reply:
I was writing this in a "just a quick 10-line script" mindset, which it was time to break out of. 7a74588